zerver/models/users.py

```diff
@@ -116,6 +116,20 @@ class UserBaseSettings(models.Model):
 
     web_mark_read_on_scroll_policy = models.SmallIntegerField(default=MARK_READ_ON_SCROLL_ALWAYS)
 
+    # UI setting controlling if clicking on a channel link should open
+    # the channel feed (interleaved view) or narrow to the first topic
+    # in the channel.
+
+    NARROW_TO_FIRST_TOPIC_IN_CHANNEL = 1
+    NARROW_TO_CHANNEL_FEED = 2
+
+    WEB_CHANNEL_DEFAULT_VIEW_CHOICES = [
+        NARROW_TO_FIRST_TOPIC_IN_CHANNEL,
+        NARROW_TO_CHANNEL_FEED,
+    ]
+
+    web_channel_default_view = models.SmallIntegerField(default=NARROW_TO_FIRST_TOPIC_IN_CHANNEL)
```

Comment:
Maybe worth checking in #backend, but I think we should use the `db_default` option to make the migration less disruptive. See `git grep db_default` for an existing example.

Reply:
Started discussion in https://chat.zulip.org/#narrow/stream/3-backend/topic/db_default.20vs.20default.20for.20Django.20models